### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by mefarazath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
 ---
 
+This repository contains the core components and features required by the [WSO2 Identity Server](https://github.com/wso2/product-is) product.
+ 
+
+## Building from the source
+
+If you want to build **carbon-identity-framework** from the source code:
+
+1. Install Java 8(http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
```
Comment (madurangasiriwardena) [inline README.md]: We can build this repo with java 7 also.

Comment (mefarazath) [inline README.md]: Changed it as Java7 or above...

File: README.md [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+1. Install Java7 or above
+1. Install Apache Maven 3.x.x(https://maven.apache.org/download.cgi#)
+1. Get a clone or download the source from this repository (https://github.com/wso2/carbon-identity-framework.git)
+1. Run the Maven command ``mvn clean install`` from the ``carbon-identity-framework`` directory.
+
+
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/carbon-identity-framework/badge/icon)](https://wso2.org/jenkins/job/carbon-identity-framework) |
+| master      | [![Build Status](https://wso2.org/jenkins/job/platform-builds/job/carbon-identity-framework/badge/icon)](https://wso2.org/jenkins/job/platform-builds/job/carbon-identity-framework/) |
```
Comment (ruwanta) [inline README.md]: @madurangasiriwardena The Jenkins badge should go just after the title, not the bottom of the README.

Comment (mefarazath) [inline README.md]: And do we need branch --> build status?

Comment (ruwanta) [inline README.md]: @mefarazath Yes, branch --> build status  would be a good addition

Comment (mefarazath) [inline README.md]: It's already there. I was wondering whether we need to remove it :smile: 
Fixed the badge position

**Pull request merged**